vue-frontend/src/components/home-new/ServiceSection.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="md:tw-pt-[100px]">
+    <div
+      class="tw-flex tw-justify-center sm:tw-py-12 tw-py-9 tw-font-roboto-bold tw-font-[700] tw-leading-[4.469rem] sm:tw-text-[3.438rem] tw-text-[2.188rem]"
+    >
+      How We Can Help You
+    </div>
+
+    <div
+      v-for="(service, index) in services"
+      :key="service"
+      @mouseover="activeIndex = index"
+      @mouseleave="activeIndex = null"
+      @touchstart.passive="activeIndex = index"
+      :class="
+        activeIndex === index
+          ? 'hover:tw-text-transparent hover:tw-shadow-[0px_2px_50px_4px_rgba(61,61,61,0.1)] tw-ease-in-out '
+          : ''
+      "
+      class="tw-relative md:tw-h-32 lg:tw-h-28 tw-w-full tw-text-[1.1rem] sm:tw-text-[1.5625rem]"
+    >
+      <div class="tw-ml-0 tw-w-full tw-text-[1.2rem] lg:tw-text-[1.375rem]">
+        <div
+          class="tw-flex tw-basis-2/4 tw-pl-[1rem] xl:tw-pl-[4.563rem] 2xl:tw-pl-[12.563rem] tw-justify-right tw-items-center tw-font-inter-regular"
+        >
+          <span class="sm:tw-py-9 tw-py-3"> 0{{ index + 1 }} </span>
+          <span class="sm:tw-p-9 tw-p-3">
+            {{ service.title }}
+          </span>
+        </div>
+      </div>
+
+      <div
+        v-if="activeIndex === index"
+        class="tw-absolute tw-flex md:tw-flex-row tw-flex-col tw-ml-0 tw-left-0 tw-top-0"
+      >
+        <div
+          :class="[service.className]"
+          class="tw-flex tw-basis-[45%] md:tw-h-32 lg:tw-h-28 tw-items-center tw-bg-gradient-to-r tw-from-[#F2709C] tw-to-[#FF835B] tw-pl-[1rem] xl:tw-pl-[4.563rem] 2xl:tw-pl-[12.563rem] tw-justify-right tw-font-inter-semibold tw-text-white tw-text-[1.375rem] lg:tw-text-[1.875rem] tw-leading-[1.875rem] lg:tw-leading-[2.4375rem]"
+        >
+          <span class="sm:tw-py-9 tw-py-3"> 0{{ index + 1 }} </span>
+          <span class="sm:tw-p-9 tw-p-3">
+            {{ service.title }}
+          </span>
+        </div>
+        <div
+          class="tw-border-l-0 tw-border-r-[50px] tw-border-solid tw-border-transparent md:tw-border-t-[128px] lg:tw-border-t-[112px] tw-border-t-[#ff835b]"
+        ></div>
+        <div
+          :class="[service.basis]"
+          class="tw-flex tw-flex-col tw-px-4 tw-basis-2/4 md:tw-basis-[60%] 2xl:tw-basis-2/4 tw-justify-center tw-items-center tw-font-inter-regular tw-text-[1rem] lg:tw-text-[1.25rem] tw-leading-[1.375rem] xl:tw-leading-[1.775rem] tw-text-black-core/[.8]"
+        >
+          <p>
+            {{ service.content }}
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+export default {
+  data() {
+    return {
+      activeIndex: null,
+      services: [
+        {
+          title: "Mobile app Development",
+          className: "md:tw-basis-[50%]",
+          basis: "md:tw-basis-[45%] 2xl:tw-basis-2/4",
```

Comment:
I think there is no need of this classes

Reply:
its needed because the text is dispalying in 2 parts .